### ISSUE TITLE
Metric missing, wrong label and not understanding ring implementation

### ISSUE THREAD

**Issue by wiardvanrij** [grafana/loki#4888]
**Describe the bug**
Setup a stack with the following components; 
- distributor
- ingester
- querier
- compactor

**Expected behavior**

`cortex_ring_members` metric is present **per** 'microservice' i.e. component. 
This is the case for all the components except the ingester. The ingester (`-target=ingester`) does not have the metric `cortex_ring_members`. 

What I also don't really understand is that `memberlist_client_cluster_members_count` would give the sum of all your components, however the `cortex_ring_members` only has ingesters and compactor as name values. Shouldn't all the components be part of that metric (with different names of the component)?  

**tl;dr**
I would like to have an overview (via metrics and maybe UI) of basically the entire ring (memberlist?) and it's ring's (per component?)

**Environment:**
 - Infrastructure: Kubernetes
 - grafana/loki:2.4.1

-----

To add a little extra; 
I'm also not completely understanding the ring mechanics/logic. Perhaps it would be nice to include some more documentation on this, as it might provide a lot more information in setting up the stack. Basically my/the question is:  Is there one ring with segments per component, or multiple rings for each component?  
If it's one ring, why not make it way more obvious that's its actually just one ring (as `memberlist_client_cluster_members_count` would suggest) with segmentation per component.

Like via the localhost/ring on the **distributor**, you see the ring for the **ingesters** (mind blown :D), but not it's own ring and the ingesters do not have a /ring page? 

All with all, its just not consistent and I honestly think it should not be this complicated if we just fix a few things and provide good context.

**Comment by wiardvanrij** [comment]
Still relevant

**Comment by Hammond95** [comment]
The metric is misleading I think. On cortex website it is defined as follows:

```
cortex_ring_members
How many ingesters can be seen in each state, by other components.
```
https://cortexmetrics.io/docs/guides/ingesters-rolling-updates/
